app/src/main/res/values/strings.xml

```diff
@@ -1800,4 +1800,5 @@
     <!-- Snackbar button text to navigate to telemetry settings.-->
     <string name="experiments_snackbar_button">Go to settings</string>
     <string name="firefox_suggest_header">Firefox Suggest</string>
+    <string name="sync_cfr_message">Your tabs are syncing! Pick up where you left off on your other device.</string>
```

Comment:
We actually want to group strings under certain headers if there is a commonality between them. Just easier to see all related strings for a feature or screen altogether. Other than we also need to provide a localization description so the localization team and translator understands where and how the string is used. 

See https://github.com/mozilla-mobile/fenix/pull/26226/files#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103 for an example.